docs/mddocs/Quickstart/ollama_portable_zip_quickstart.md

```diff
@@ -206,9 +208,9 @@ To enable `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS`, set it **before start
 > [!TIP]
 > You could refer to [here](https://www.intel.com/content/www/us/en/developer/articles/guide/level-zero-immediate-command-lists.html) regarding more information about Level Zero Immediate Command Lists.
 
-### Additional models supported after Ollama v0.5.4
```

Comment:
Should we directly remove this section? As some of the following models may be already supported in Ollama 0.6.2, so they are not "additional models"